.travis.yml

```diff
@@ -4,10 +4,19 @@ node_js:
     - "0.12"
     - "4.2"
 
-sudo: false
+sudo: required
 
-services:
-    - cassandra
+before_install:
+  - sudo rm -rf /var/lib/cassandra/*
+  - sudo wget -O /var/lib/cassandra/cassandra.tar.gz https://archive.apache.org/dist/cassandra/2.1.8/apache-cassandra-2.1.8-bin.tar.gz
+  - sudo tar -xzf /var/lib/cassandra/cassandra.tar.gz -C /var/lib/cassandra
```

Comment:
`/var/lib/cassandra` is the default path where Cassandra is putting its data files, so perhaps a different location would be more appropriate? `/var/lib/cassandra-code` ?